Source/ACE.Server/WorldObjects/Corpse.cs

```diff
@@ -225,8 +225,8 @@ public void GenerateRare(WorldObject killer)
             var tier = LootGenerationFactory.GetRareTier(wo.WeenieClassId);
             LootGenerationFactory.RareChances.TryGetValue(tier, out var chance);
 
-            log.Info($"[RARE] {Name} ({Guid}) generated rare {wo.Name} ({wo.Guid}) for {killer.Name} ({killer.Guid})");
-            log.Info($"[RARE] Tier {tier} -- 1 / {chance:N0} chance");
+            log.Debug($"[RARE] {Name} ({Guid}) generated rare {wo.Name} ({wo.Guid}) for {killer.Name} ({killer.Guid})");
```

Comment:
most definitely not debug messages.

Reply:
Sever operators may want to see this in their console output, we should ask.